src/base/utils.js

```diff
@@ -28,7 +28,16 @@ var formatTime = function(time) {
     var minutes = time % 60
     time = parseInt(time/60)
     var hours = time % 24
+    var days = parseInt(time/24)
     var out = ""
+<<<<<<< HEAD
+    if (days && days > 0) {
+      out += days + ":"
+      if (hours < 1) out += "00:"
+    }
+=======
```

Comment:
Expected an identifier and instead saw '==='.<br>Expected an operator and instead saw '==='.<br>Expected an operator and instead saw '='.